db/schema.rb

```diff
@@ -638,6 +638,7 @@
     t.boolean  "exporting_photos",                                 default: false
     t.string   "color_theme",                        limit: 255
     t.boolean  "post_default_public",                              default: false
+    t.boolean  "chat_auto_login",                                  default: false
```

Comment:
Oh sorry, I misunderstood. I thought from your comment that you meant that every member of a pod will be automatically logged into chat when they next sign in to the pod after the software update, regardless of whether they wanted to use chat.

You're talking about only the people who were signed into chat when they signed out of diaspora\*. That's fine, of course.